### DISCUSSION TITLE
Beta 13

### DISCUSSION THREAD

**Discussion by stohrendorf** [stohrendorf/CroftEngine#206] [General]
⚠️ Savegames are not compatible with previous versions.

## Fixes
* inventory rendering fixed (EE-423)
* drawing a weapon that's not in inventory doesn't produce a crash anymore
* shotguns do more damage now (EE-434)
* boulder doesn't roll of to one side anymore (EE-433)
* t-rex doesn't turn head too much anymore (EE-438)
* bats should not shiver anymore when dying and hitting the ground (EE-442)
* winged mutant now flies properly now (EE-437, EE-395)
* boulders now fully drop to the ground when stopped (EE-310)
* some textures didn't animate properly (EE-443)

## New Features
* lighing added to inventory (EE-422)
* ambient track volume fading (EE-426)
* pc cheats to skip level and add all weapons
* lara says "no" when using a key/puzzle item in the inventory (EE-415)
* page indicator added to savegame list (EE-427)
* savegames can be sorted by date now (use step left/right) (EE-430)

## Improvements
* logarithmic instead of linear volume
* save slots increased to 100 (EE-428)
* picking up weapons now shows both weapon and ammo widgets (EE-436)
* faster ai path calculations
* no more showing the console window in release builds (EE-439)
* crumbling floors shouldn't show rendering issues anymore (EE-445)

## Changes
* scripts cleaned up (EE-421)
* most shadows removed so only "living" entities cast shadows (EE-435)
* dust particles darkened, reduced in size, randomized size, and increased density

<hr /><em>This discussion was created from the release <a href='https://github.com/stohrendorf/EdisonEngine/releases/tag/beta-13'>Beta 13</a>.</em>

**Comment by PittStone** [comment]
An Option for activate V-sync would be nice. I see tearing in the over half of the Screen. I can activate it with my Nvidia Options but i have a pc with a intel HD600 and that Option not exists.

**Reply by stohrendorf** [reply]
vsync is enabled by default. Are you using Windows 11? I have a report of tearing, and the only suspect was Windows 11. If I remember correctly, the workaround was to explicitly enable vsync in the driver.

**Reply by PittStone** [reply]
No its Windows 10 with newest Updates and H2.

**Reply by stohrendorf** [reply]
Unfortunately I can't help you here, as I don't know why vsync should be turned off, except the graphics driver explicitly disables it. It might be I have a bug somewhere, but as vsync seems to work for most of the people, I'm not looking into this with high priority.

**Reply by stohrendorf** [reply]
From what I could find on the internet, vsync (from the application's point of view) is working fine, it's the driver that de-syncs the synced rendering and the screen refresh, the only (more or less) reliable way to solve this is to explicitly enable vsync in the driver. I haven't found anything yet to control that from within the application.
